web/index.html

```diff
@@ -38,9 +38,22 @@
 </head>
 <body>
 
-<main>
+<header>
```

Comment:
Now that we're adding an element outside of `main`, we need to tweak the "inert" handling...

1. In styles.css, we need to [update this line](https://github.com/getify/youperiod.app/blob/3ed8cde8412b073db441ed5be733504c5bd82f43/web/css/style.css#L42) to include `header[inert]` -- and while we're at it, `footer[inert]`.
2. In `notification-manager`, we need to update the setting/clearing of `inert` on [this line](https://github.com/getify/youperiod.app/blob/3ed8cde8412b073db441ed5be733504c5bd82f43/web/js/notification-manager.js#L51) and [this line, too](https://github.com/getify/youperiod.app/blob/3ed8cde8412b073db441ed5be733504c5bd82f43/web/js/notification-manager.js#L117)
3. However, we will also need to tweak how [`NotificationManager.init()`](https://github.com/getify/youperiod.app/blob/3ed8cde8412b073db441ed5be733504c5bd82f43/web/js/notification-manager.js#L16) works, because right now it only receives and saves a single reference to the `main` DOM element, and now we'll need to access all 3 (`header`, `main`, and `footer`). Perhaps the best thing is to pass in just a reference to `body` and have notification-manager pull out and save the 3 DOM element references (via `querySelector("main")`, etc).